dbt_platform_helper/commands/pipeline.py

```diff
@@ -170,3 +204,44 @@ def _create_file_from_template(
     ).render(template_data)
     message = mkfile(base_path, pipelines_dir / file_name, contents, overwrite=True)
     click.echo(message)
+
+
+def _generate_terraform_environment_pipeline_manifest(
+    application,
+    aws_account,
+    cli_terraform_platform_modules_version,
+    platform_config_terraform_modules_default_version,
+    deploy_branch,
+):
+    env_pipeline_template = setup_templates().get_template("environment-pipelines/main.tf")
+
+    terraform_platform_modules_version = _determine_terraform_platform_modules_version(
+        cli_terraform_platform_modules_version, platform_config_terraform_modules_default_version
+    )
+
+    contents = env_pipeline_template.render(
+        {
+            "application": application,
+            "aws_account": aws_account,
+            "terraform_platform_modules_version": terraform_platform_modules_version,
+            "deploy_branch": deploy_branch,
+        }
+    )
+
+    dir_path = f"terraform/environment-pipelines/{aws_account}"
+    makedirs(dir_path, exist_ok=True)
+
+    click.echo(mkfile(".", f"{dir_path}/main.tf", contents, overwrite=True))
+
+
+def _determine_terraform_platform_modules_version(
```

Comment:
Could this be extracted to utlis? I think this is repeated in the environment command too